docs/source/quantization.rst

```diff
@@ -35,6 +35,13 @@ that perform all or part of the computation in lower precision. Higher-level
 APIs are provided that incorporate typical workflows of converting FP32 model
 to lower precision with minimal accuracy loss.
 
+Quantization requires users to be aware of three concepts:
+1. Quantization Config (Qconfig): Specifies how weights and activations are to be quantized. Qconfig is needed to create a quantized model.
+2. Backend: Refers to kernels that support quantization, usually with different numerics.
```

Comment:
Should we add an example here with references to FBGEMM/QNNPACK perhaps?

Reply:
A few lines below, the doc talks about backends in detail.